src/Collections/Sites.php

```diff
@@ -190,41 +179,74 @@ public function findUuidByName($name)
     }
 
     /**
+     * Get the site's UUID given either it's site name or it's UUID
+     *
+     * @param $id
+     * @return string
+     */
+    public function findUuid($id)
```

Comment:
This function feels clunky and misnamed. It's kind of a UUID-verifying middleware that operates another function which does what this function says it would do. Since it is only called once, perhaps it can be removed, or the UUID-verifying portion moved into some `isValidUUID` function.